docs/assets/less/stacks-documentation.less

```diff
@@ -255,10 +269,12 @@
         box-shadow: none;
     });
 
-    > pre.s-code-block {
+    > pre.s-code-block,
+    > .stacks-clipboard-content pre.s-code-block {
         border-radius: var(--br-md) var(--br-md) 0 0;
         border: 1px solid var(--bc-medium);
         max-height: 24rem;
+        padding-right: var(--su32);
```

Comment:
When you scroll all the way to the right, we want to be able to see and select everything.